src/bindings/python/src/pyopenvino/frontend/tensorflow/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include(${PYTHON_SOURCE_DIR}/pyopenvino/frontend/frontend_module.cmake)
```

Comment:
I suppose should be `OpenVINOPython_SOURCE_DIR` since project is named `OpenVINOPython` (see https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/CMakeLists.txt#L7)